homeassistant/components/litterrobot/strings.json

```diff
@@ -25,6 +25,30 @@
       "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
     }
   },
+  "issues": {
+    "service_deprecation_turn_off": {
+      "title": "Litter-Robot vaccum support for vacuum.turn_off is being removed",
+      "fix_flow": {
+        "step": {
+          "confirm": {
+            "title": "[%key:component::litterrobot::issues::service_deprecation_turn_off::title%]",
+            "description": "Litter-Robot vaccum support for the vacuum.turn_off service is deprecated and will be removed in Home Assistant 2024.2; Please adjust any automation or script that uses the service to instead call vacuum.stop and select submit below to mark this issue as resolved."
```

Comment:
Best practices regarding talking about service names and using placeholders for the service names referencing the YAML English name or letting our translators translate the service name hopefully using the translated GUI name is unclear to me.